ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal

```diff
@@ -33,6 +33,7 @@ library
                        Ouroboros.Consensus.Cardano.CanHardFork
                        Ouroboros.Consensus.Cardano.Node
                        Ouroboros.Consensus.Cardano.ShelleyBased
+                       Ouroboros.Consensus.Cardano.Orphans
```

Comment:
This list should be in alphabetical order. Would you fixup the `ShellHFC` bit too? Thanks.